packages/sfui/frameworks/vue/components/SfSwitch/SfSwitch.vue

```diff
@@ -40,5 +42,6 @@ const proxyChecked = computed({
     type="checkbox"
     role="switch"
     data-testid="switch"
+    :aria-checked="switchRef?.checked"
```

Comment:
Question: why cannot we go with?
```suggestion
    :aria-checked="proxyChecked"
```

Reply:
`proxyChecked` is model that contain multiple input values, what I need here is this particular `input` check value

Reply:
Right! It would be great to have a comment about this somewhere here, this part of vue API might be tricky for some 😄

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2915/commits/3f04e29c6dbe760abf8ecf5340d95d374de8c44e